### PR TITLE
Updating publish instructions

### PR DESCRIPTION
**Pull request by aaronsarnat**
This PR adds additional detail to the publish process, and makes a minor correction with the execution command.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hierplane",
-  "version": "0.0.10",
+  "version": "0.0.11",
```
Comment (vidurj) [inline package.json]: Why did this change?

**Review by vidurj — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (aaronsarnat) [reply package.json]: actually it shouldn't have. this was a mistake. thanks for reminding me... i need to revert this

Comment (aaronsarnat) [reply package.json]: done

**Comment by aaronsarnat** [comment]
Got Vidur's approval, merging.

**Pull request merged**